README.md

```diff
@@ -8,4 +8,21 @@ An example of crate usage can be found in [`src/main.rs`](src/main.rs)
 Requirements
 ------------
 
-Building this (and the dependent crates) requires a [nightly compiler](https://github.com/rust-lang-nursery/rustup.rs/blob/master/README.md#working-with-nightly-rust).
+Building this (and the dependent crates) requires Rust v1.30 or later.
+
+Setup
+------------
+### We need to we need to get OpenOCD on our machine. If you already have it, feel free to skip this step. 
+* Install or build from source the latest version of [OpenOCD](http://openocd.org/).
+### To launch OpenOCD and target your ST NUCLEO-L031K6
+* `openocd -f interface/stlink-v2-1.cfg -f target/stm32l0.cfg`
+* NOTE: Your target `stlink.cfg` file may change depending on what version of OpenOCD you're running. 
```

Comment:
add as much detail here as you can, or at least some suggestion about how to find more info. also "your file may change" is kind of nebulous.